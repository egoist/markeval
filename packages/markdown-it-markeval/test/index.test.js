const markdown = require('markdown-it')

const md = markdown()
  .use(require('../'))

test('it renders', () => {
  const input = `
\`\`\`js
// @markeval
module.exports = 42
\`\`\`

\`\`\`js
// @markeval
module.exports = require('path').dirname('foo/bar.js')
\`\`\`

\`\`\`js
module.exports = 'lol'
\`\`\`
`
  expect(md.render(input)).toMatchSnapshot()
})
