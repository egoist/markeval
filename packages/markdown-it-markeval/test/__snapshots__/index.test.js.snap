// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
"42foo<pre><code class=\\"language-js\\">module.exports = 'lol'
</code></pre>
"
`;
